docs/python_interface.rst

```diff
@@ -262,6 +262,22 @@ Usage examples
   print("Final status:")
   print(r.stats)
 
+.. code-block:: python
+
+  import ansible_runner
+  # Specifying custom container working directory while using Podman
+  # `/tmp/demo` contains `my_project` directory which contains all the Ansible content
+  res = ansible_runner.run(
+    private_data_dir='/tmp/demo',
+    host_pattern='localhost',
+    container_workdir='/runner/my_project',
```

Comment:
Is this correct? This Issue/PR is about getting it to properly respect `project_dir`.  Is we want to add this example, maybe name it something besides `my_project` to not create confusion with a custom `project_dir` setting?